packages/pwa-buildpack/lib/cli/create-custom-origin.js

```diff
@@ -42,10 +38,9 @@ module.exports.handler = async function buildpackCli({ directory }) {
                 ports.development
             } and staging server will run on port ${ports.staging}.`
         );
-    } finally {
-        // change back if we changed directories
-        if (workDir !== projectRoot) {
-            process.chdir(workDir);
-        }
+    } catch (e) {
+        prettyLogger.error(e.message);
+        // eslint-disable-next-line no-process-exit
+        process.exit(2);
```

Comment:
Because it's an actual CLI command being handled at top level. But I think I get what you're driving at. I can handle the process breaks in the one CLI file.